### DISCUSSION TITLE
will there be a version for 1.21.1?

### DISCUSSION THREAD

**Discussion by tito323** [nonamecrackers2/crackers-wither-storm-mod#2280] [Q&A] [answered]
I would like to see a version of 1.21.1 forge. Why does it only come for 1.20.1 and not for newer versions?

**Comment by Infinitekaosv3** [comment, marked as answer]
Yes 100%

**Comment by BluePineappleGuy** [comment]
it will come soon, but the devs are probably in the middle of a new content update. both will arrive at the same time.

**Comment by Mariosonicman91** [comment]
Another quick question, will said 1.21.1 port be on Forge or NeoForge? Because Forge is dying and having the mod only on there will exclude a chunk of people. God, I hate these mod loader disputes. What was it like when it was just Forge? I'll never know, as I joined too late to experience it. Must've been nice, every mod working together, especially on 1.12.2.

**Reply by BluePineappleGuy** [reply]
i fr hate when the two mods i want to play with are for different loaders, the pain just gets worse the more loaders keep popping up. Cant even make a modpack anymore

**Reply by Mariosonicman91** [reply]
Real, I have a NeoForge 1.21.1 pack and there's quite a few modpack utility mods that'd be pretty nice 😭 e.g. Item Obliterator, Block Swap. At least Sinytra Connector is doing some damage control.

**Reply by nonamecrackers2** [reply]
It will be on NeoForge

**Reply by Mariosonicman91** [reply]
> It will be on NeoForge

Absolutely beautiful news!

**Reply by gruzzistaken** [reply]
YESSSSSSSSSSSSS LESSSSS GOOOOOO

**Reply by coreo-oreo** [reply]
> It will be on NeoForge

Sorry if this comment bothers you, but is there an ETA or not yet?

**Reply by watersofnazareth** [reply]
> > It will be on NeoForge
> 
> 
> 
> Sorry if this comment bothers you, but is there an ETA or not yet?

Next few months hopefully

**Reply by coreo-oreo** [reply]
Alright thanks!

**Reply by okil6dev** [reply]
is it comming its a year later

**Reply by Mariosonicman91** [reply]
Judging by the fact that a collaborator said it'll hopefully come in few months in July a year ago, the fact Aeronautics released and there's confirmed compatibility, I'd say it's going to be a while before we see anything. Probably a good chunk of new content, too. Not even sure what they'd add atp. Keep doing the great work devs

**Reply by orangetabby0** [reply]
I hope so I'm so genuinely hyped for this, this is going to be my GTA6 if it comes out in a few months
